### PR TITLE
add missing arguments to deploy

### PR DESCRIPTION
**Pull request by dfangshuo**
Added missing a argument to ballot/app/scripts/deploy_service.js

Ran `node app/scripts/deploy_service.js` from /ballot to and verified that the contract was successfully deployed

### CODE REVIEW

File: ballot/app/scripts/deploy_service.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const oasis = require('@oasislabs/client');
 
-oasis.workspace.Ballot.deploy({
+oasis.workspace.Ballot.deploy('pikachu',  ['snorlex', 'ghastly'], {
```
Comment (eauge) [inline ballot/app/scripts/deploy_service.js]: the first argument should be a description, the list of strings is the list of candidates

Comment (nhynes) [reply ballot/app/scripts/deploy_service.js]: I thought it was asking which was your favorite pikachu. Isn't the Ash guy a pikachu trainer?

Comment (ssavvides) [reply ballot/app/scripts/deploy_service.js]: also ... what's a `snorlex` and `ghastly`?

Comment (eauge) [reply ballot/app/scripts/deploy_service.js]: it should actually be snorlax. `ghastly` and `snorlax` are pokemons. These are the candidates we use in the secret ballot app

Comment (ssavvides) [reply ballot/app/scripts/deploy_service.js]: "snorlax" and "gastly"

Comment (eauge) [reply ballot/app/scripts/deploy_service.js]: you are correct!

Comment (dfangshuo) [reply ballot/app/scripts/deploy_service.js]: updated description to conform to this pattern!

File: hello-world/app/scripts/deploy_service.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const oasis = require('@oasislabs/client');
 
-oasis.workspace.HelloWorld.deploy({
+oasis.workspace.HelloWorld.deploy('pikachu',  ['snorlex', 'ghastly'], {
```
Comment (ssavvides) [inline hello-world/app/scripts/deploy_service.js]: `HelloWorld` does not need any arguments, does it?

The arguments in `deploy` should match the constructor (`new`) in the service, minus the `Context`: https://github.com/oasislabs/tutorials/blob/master/hello-world/service/src/main.rs#L20

File: ballot/app/scripts/deploy_service.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const chalk = require('chalk');
 const oasis = require('@oasislabs/client');
 
-oasis.workspace.Ballot.deploy({
+oasis.workspace.Ballot.deploy('pokemon',  ['snorlex', 'ghastly'], {
```
Comment (nhynes) [inline ballot/app/scripts/deploy_service.js]:
```suggestion
oasis.workspace.Ballot.deploy('pokemon',  ['snorlax', 'ghastly'], {
```

File: hello-world/app/scripts/deploy_service.js (hunk 2 above)
Comment (nhynes) [inline hello-world/app/scripts/deploy_service.js]:
```suggestion
oasis.workspace.HelloWorld.deploy({
```

File: ballot/app/scripts/deploy_service.js (hunk 1 above)
Comment (ssavvides) [reply ballot/app/scripts/deploy_service.js]: ty. also change the names of the pokemon, per the discussion above.

File: hello-world/app/scripts/deploy_service.js (hunk 2 above)
Comment (dfangshuo) [reply hello-world/app/scripts/deploy_service.js]: you're right - accidentally made this change, reverted it!

**Pull request closed without merging**